requirements/test.txt

```diff
@@ -6,6 +6,7 @@ pytest-django==4.4.0
 pytest-mock==3.6.1
 pytest-deadfixtures==2.2.1
 pytest-randomly==3.10.1
+pytz==2021.3
```

Comment:
Is this related to Django 4.0 release deprecated pytz?

Reply:
I think so

Reply:
@maxipavlovic i suggest not to change test requirements now and add Django 4 support in separate task.
Do you have any plans for this?

Reply:
I think it's ok to change test requirements in this task, but can be done separately. As you wish :)